webapp/channels/src/components/admin_console/billing/billing_summary/upsell_card.scss

```diff
@@ -1,69 +0,0 @@
-@import 'utils/mixins';
-
-.UpsellCard {
```

Comment:
We don't need this anymore? Seems we still have the `UpsellCard` component

Reply:
UpsellCard isn't used anywhere, must have missed deleting the component itself